nbbuild/licenses/MIT-winp

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
This file seems unnecessary, I've started a thread on dev@ to talk about this https://lists.apache.org/thread.html/b86dd173dd91f27d6e6b48c429f03bca5afe8e417fdbea4e9484da37@%3Cdev.netbeans.apache.org%3E